pages/view/_iscnId.vue

```diff
@@ -191,41 +390,80 @@ export default class ViewIscnIdPage extends Vue {
     return this.record.stakeholders
   }
 
-  created() {
-    const { iscnId } = this.$route.params
-    if (iscnId.startsWith(ISCN_PREFIX)) {
-      this.iscnId = iscnId
-    }
+  get transactionsURL() {
+    return `${BIG_DIPPER_TRANSACTIONS}${this.txHash}`
   }
 
-  async mounted() {
-    if (!this.iscnId) {
-      const param = this.$route.params.iscnId
-      if (!isCosmosTransactionHash(param)) {
-        this.$nuxt.error({
-          statusCode: 400,
-          message: ErrorMessage.statusCode400,
-        })
-        return
-      }
-      const res = await this.fetchISCNByTx(param)
-      if (!res) {
-        this.$nuxt.error({
-          statusCode: 400,
-          message: ErrorMessage.statusCode400,
-        })
-        return
-      }
-      this.iscnId = res.records[0].id
-      this.$router.replace({ params: { iscnId: this.iscnId } })
+  get rawDataURL() {
+    return `${ISCN_TX_RAW_DATA_ENDPOINT}${this.iscnId}`
```

Comment:
Should use ISCN query by ID endpoint `/iscn/records/id?iscn_id=${iscnId}` if you do not want the transaction hash, only the ISCN record data